src/lsm/fuzz.zig

```diff
@@ -0,0 +1,467 @@
+const std = @import("std");
+const testing = std.testing;
+const allocator = testing.allocator;
+const assert = std.debug.assert;
+const os = std.os;
+
+const config = @import("../config.zig");
+const vsr = @import("../vsr.zig");
+const log = std.log.scoped(.lsm_forest_test);
+
+const MessagePool = @import("../message_pool.zig").MessagePool;
+const Transfer = @import("../tigerbeetle.zig").Transfer;
+const Account = @import("../tigerbeetle.zig").Account;
+const Storage = @import("../test/storage.zig").Storage;
+const StateMachine = @import("../state_machine.zig").StateMachineType(Storage);
+
+const GridType = @import("grid.zig").GridType;
+const GrooveType = @import("groove.zig").GrooveType;
+const Forest = StateMachine.Forest;
+
+const Grid = GridType(Storage);
+const SuperBlock = vsr.SuperBlockType(Storage);
+
+const FuzzOp = union(enum) {
+    compact,
+    put: Account,
+    get: struct {
+        id: u128,
+    },
+};
+
+const Environment = struct {
+    const cluster = 32;
+    const replica = 4;
+    // TODO Is this appropriate for the number of fuzz_ops we want to run?
+    const size_max = vsr.Zone.superblock.size().? + vsr.Zone.wal.size().? + (512 + 64) * 1024 * 1024;
+
+    const node_count = 1024;
+    const cache_entries_max = 2 * 1024 * 1024;
+    const forest_options = StateMachine.forest_options(.{
+        // Ignored by StateMachine.forest_options().
+        .lsm_forest_node_count = undefined,
+        .cache_entries_accounts = cache_entries_max,
+        .cache_entries_transfers = cache_entries_max,
+        .cache_entries_posted = cache_entries_max,
```

Comment:
Unless you want to avoid permuting any seeds you've found, it would make sense to either drop the cache size, or increase the range of keys being inserted, to ensure that the whole dataset can't fit in the cache.

Reply:
Done. 

I'll have to add prefetching too.